packages/accordion/test/accordion.test.ts

```diff
@@ -15,11 +15,7 @@ import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
 import '../sp-accordion.js';
 import { Default } from '../stories/accordion.stories.js';
 import { Accordion, AccordionItem } from '@spectrum-web-components/accordion';
-import {
```

Comment:
previously the accordion supported both Tab AND up/down arrows. I'm absolutely fine with just Tab/Shift-Tab being the end result but want to make sure that the change was intentional

Reply:
I think up/down, when pressed while an accordion header button is focused, makes a lot of sense for quickly navigating - especially navigating _past_ large sets of focusable content to a sibling header, as in [this example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/accordion/accordion.html).

This PR already moves accordion into a more-accessible state, so I recommend merging it & then adding support for arrow keys if you decide to as a separate endeavor.